Fix deleted default constructor warning

**Describe the bug**

```
20:39:29  [252/1431] Building CXX object TPL/vt/src/CMakeFiles/vt.dir/vt/messaging/active.cc.o
20:39:29  FAILED: TPL/vt/src/CMakeFiles/vt.dir/vt/messaging/active.cc.o
20:39:29  /opt/atse/mpi/openmpi4-arm/4.0.2/bin/mpicxx  -DDISABLE_TPL_CHECKPOINT=0 -DEMPIRE_CHECK_FPE -DFMT_HEADER_ONLY=1 -DFMT_USE_USER_DEFINED_LITERALS=0 -DHAS_DETECTION_COMPONENT=1 -DKOKKOS_ENABLED_CHECKPOINT=1 -DKOKKOS_KERNELS_ENABLED=1 -I/opt/atse/libs/arm/zlib/1.2.11/include -I/lustre/empire-jenkins/stria/workspace/Deploy-develop-stria-arm-20-opt-openmp-static/src/EMPIRE/TPL/vt/lib/fmt -I/lustre/empire-jenkins/stria/workspace/Deploy-develop-stria-arm-20-opt-openmp-static/src/EMPIRE/TPL/vt/lib/CLI -ITPL/vt/release -I/lustre/empire-jenkins/stria/workspace/Deploy-develop-stria-arm-20-opt-openmp-static/src/EMPIRE/TPL/vt/src -ITPL/checkpoint/src -I/lustre/empire-jenkins/stria/workspace/Deploy-develop-stria-arm-20-opt-openmp-static/src/EMPIRE/TPL/checkpoint/src -I/lustre/empire-jenkins/stria/workspace/Deploy-develop-stria-arm-20-opt-openmp-static/src/EMPIRE/TPL/detector/src -isystem /projects/empire/installs/stria/ARM-20.0_OPENMPI-4.0.2-RELEASE-OPENMP-STATIC/trilinos/2021-01-12_01.00.11.192908/lib/cmake/Kokkos/../../../include -fPIC -fopenmp=libomp -O3 -DNDEBUG -Wall -Wshadow -Wunused-variable -pedantic -Wno-unknown-pragmas -Wno-narrowing -Wno-format -Werror -DTEUCHOS_DEPRECATED= -DTPETRA_DEPRECATED= -O3 -DNDEBUG   -std=c++1y -ftemplate-depth=900 -DCLI11_EXPERIMENTAL_OPTIONAL=0 -fPIC -fopenmp=libomp -std=c++14 -MD -MT TPL/vt/src/CMakeFiles/vt.dir/vt/messaging/active.cc.o -MF TPL/vt/src/CMakeFiles/vt.dir/vt/messaging/active.cc.o.d -o TPL/vt/src/CMakeFiles/vt.dir/vt/messaging/active.cc.o -c /lustre/empire-jenkins/stria/workspace/Deploy-develop-stria-arm-20-opt-openmp-static/src/EMPIRE/TPL/vt/src/vt/messaging/active.cc
20:39:29  /lustre/empire-jenkins/stria/workspace/Deploy-develop-stria-arm-20-opt-openmp-static/src/EMPIRE/TPL/vt/src/vt/messaging/active.cc:181:3: error: explicitly defaulted default constructor is implicitly deleted [-Werror,-Wdefaulted-function-deleted]
20:39:29    MultiMsg() = default;
20:39:29    ^
20:39:29  /lustre/empire-jenkins/stria/workspace/Deploy-develop-stria-arm-20-opt-openmp-static/src/EMPIRE/TPL/vt/src/vt/messaging/active.cc:193:12: note: default constructor of 'MultiMsg' is implicitly deleted because field 'info' has no default constructor
20:39:29    SendInfo info;
20:39:29             ^
20:39:29  1 error generated.
```

Discussion at https://github.com/DARMA-tasking/vt/pull/1209/files#diff-4046ba3c8683291d70d6f122814d8983e2b3d6dbca6fa7f79f755745599e13ceR55

**To Reproduce**
Steps to reproduce the behavior:
1. Example/test/snippet of code that fails
2. Compiler, platform, libraries
3. Run command: number of processors, threading options, etc.
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Platform (please complete the following information):**
 - Supercomputer or desktop [e.g., Trinity, Sierra)
 - Compiler environment [e.g., clang-3.9, gcc8]
 - Libraries [e.g., mpich-2, openmpi)
 - Commit ID or release of VT [e.g., vt-1.0.0]

**Additional context**
Add any other context about the problem here.

--- a/src/vt/messaging/active.cc
+++ b/src/vt/messaging/active.cc
@@ -178,7 +178,6 @@ EventType ActiveMessenger::sendMsgBytesWithPut(
 }
 
 struct MultiMsg : vt::Message {
-  MultiMsg() = default;
   MultiMsg(SendInfo in_info, NodeType in_from, MsgSizeType in_size)
     : info(in_info),
       from(in_from),

